### PR TITLE
[April fools] Raises the roleplay level by adding athletics skill and cars.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
 add: Everyone is now expected to train to get their murderbone walk speed. Hit the gym. You start slower now.
add: Everyone now starts with a car beacon that can be used to receive a complimentary NT vehicle. These were made from scraps of materials NT had left over
/:cl:

this raises rp level. no more run and gunning

### CODE REVIEW

**Comment by TiviPlus** [comment]
YOU CANT JUST POLLUTE THE STATION BY GIVING EVERY ONE A COMPLIMENTARY CAR NOOOOOO
Haha car go brr

File: code/modules/vehicles/cars/car.dm [hunk 1]
```diff
@@ -93,3 +94,103 @@
...
+	. = ..()
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(!(L.mobility_flags & MOBILITY_STAND) && !L.buckle_lying)
+		for(var/i in return_drivers())
+			if(!ishuman(i))
+				continue
+			var/mob/living/carbon/human/boomer = i
+			boomer.say(pick("DON'T LIE INFRONT OF MY CAR YA FUCKIN KNOB", "GET OUTA DA WAY I'M DRIVIN' HERE", "YOU ARE RUINING MY SUSPENSION CUNT!"), forced="car crash")
```
Comment (trollbreeder) [inline code/modules/vehicles/cars/car.dm]: check for age first, say boomer lines if old enough, if young instead say something post-millenials would say

**Comment by trollbreeder** [comment]
Grand theft auto 13

**Comment by HomocidlePancakes** [comment]
Time to gangbang. DORMS STREET 4 LYFE

**Comment by zxaber** [comment]
If we're not careful, we're gonna accidentally make a better game exclusively for April 1st.

File: code/modules/vehicles/cars/car.dm [hunk 2]
```diff
@@ -93,3 +96,117 @@
 		M.visible_message("<span class='warning'>[M] is forced into \the [src]!</span>")
 	M.forceMove(src)
 	add_occupant(M, VEHICLE_CONTROL_KIDNAPPED)
+
+
+/obj/vehicle/sealed/car/civ
+	name = "car"
+	desc = "This used to be a pedastrian focused station, and now all these cars are ruining it!." //ok boomer
```
Comment (cacogen) [inline code/modules/vehicles/cars/car.dm]:
```suggestion
	desc = "This used to be a pedestrian-focused station, and now all these cars are ruining it!" //ok boomer
```
i know this is for april fools so shittiness is part and parcel but... also boomers don't care about the environment

Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: back in their days everyone rode horses

File: code/modules/vehicles/cars/car.dm [hunk 3]
```diff
@@ -93,3 +96,117 @@
...
+	. = ..()
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(!(L.mobility_flags & MOBILITY_STAND) && !L.buckle_lying)
+		for(var/i in return_drivers())
+			if(!ishuman(i))
+				continue
+			var/mob/living/carbon/human/boomer = i
+			boomer.say(pick("DON'T LIE INFRONT OF MY CAR YA FUCKIN KNOB", "GET OUTA DA WAY I'M DRIVIN' HERE", "YOU ARE RUINING MY SUSPENSION CUNT!"), forced="car crash")
```
Comment (cacogen) [inline code/modules/vehicles/cars/car.dm]:
```suggestion
			boomer.say(pick("DON'T LIE IN FRONT OF MY CAR YA FUCKIN' KNOB!!", "HEY! I'M DRIVIN' HERE!!", "YOU ARE RUINING MY SUSPENSION, CUNT!"), forced="car crash")
```
i'm drivin here may as well be a reference to midnight cowboy

Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: what now

Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: oh

File: code/modules/vehicles/cars/car.dm [hunk 4]
```diff
@@ -93,3 +96,117 @@
...
+/obj/vehicle/sealed/car/civ/Bump(atom/A)
+	. = ..()
+	if(!isliving(A))
+		return
+	var/mob/living/L = A
+	for(var/i in return_drivers())
+		if(!ishuman(i))
+			continue
+		var/mob/living/carbon/human/boomer = i
+		boomer.say(pick("HOLY SHIT MY PAINT IS RUINED", "I LEASED THIS YOU DICK!", "I WILL END YOU GET OUT OF THE WAY!", "WHAT THE FUCK DO YOU KNOW HOW EXPENSIVE CARS ARE?", "GET OF THE ROAD YOU BRAINDEAD TROGLODYTE", "GET OUT OF THE GODDAMN WAY"), forced="car crash")
```
Comment (cacogen) [inline code/modules/vehicles/cars/car.dm]:
```suggestion
		boomer.say(pick("HOLY SHIT MY PAINT IS RUINED!!", "I LEASED THIS YOU DICK!!", "GET OUT OF THE WAY, I WILL END YOU!!", "WHAT THE FUCK!? DO YOU KNOW HOW EXPENSIVE CARS ARE?!!", "GET OUT OF THE ROAD YOU BRAINDEAD TROGLODYTE!!", "GET OUT OF THE GODDAMN WAY!!"), forced="car crash")
```

Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: it's get off the road i typo'd it, get out of the road sounds odd

Comment (cacogen) [reply code/modules/vehicles/cars/car.dm]: the americans say it

Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: wtf

Comment (cacogen) [reply code/modules/vehicles/cars/car.dm]: i changed it

**Comment by oklews** [comment]
Cars for everyone, on maint city? Has science gone too far?

File: code/modules/vehicles/cars/car.dm [hunk 5]
```diff
@@ -93,3 +96,116 @@
...
+	enter_delay = 50 //No speed escapes
+	max_occupants = 5
+	movedelay = 1.15
+	material_flags = MATERIAL_AFFECT_STATISTICS | MATERIAL_COLOR | MATERIAL_ADD_PREFIX
+	var/obj/item/card/id/linked_id = null
+	var/can_drive = FALSE
+
+
+/obj/vehicle/sealed/car/civ/Initialize()
+	var/datum/material/M = pick(subtypesof(/datum/material))
```
Comment (ATH1909) [inline code/modules/vehicles/cars/car.dm]: hm yes today I will ram people with my +9 car of the heavens

File: code/modules/vehicles/cars/car.dm [hunk 6]
```diff
@@ -93,3 +96,116 @@
...
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(!(L.mobility_flags & MOBILITY_STAND) && !L.buckle_lying)
+		for(var/i in return_drivers())
+			if(!ishuman(i))
+				continue
+			var/mob/living/carbon/human/boomer = i
+			boomer.say(pick("DON'T LIE IN FRONT OF MY CAR YA FUCKIN' KNOB!!", "HEY! I'M DRIVIN' HERE!!", "YOU ARE RUINING MY SUSPENSION, CUNT!"), forced="car crash")
+		L.adjustBruteLoss(3) //tires burn
```
Comment (ATH1909) [inline code/modules/vehicles/cars/car.dm]: c'mon, make this (and bumping into people with a car) deal more damage, I know you want to

Comment (trollbreeder) [reply code/modules/vehicles/cars/car.dm]: based on the mulebot damage when running people over
```suggestion
		L.adjustBruteLoss(50) //tires burn
```

File: code/modules/vehicles/cars/car.dm (hunk 5 above)
Comment (trollbreeder) [reply code/modules/vehicles/cars/car.dm]: Obama car obama car obama car

Comment (ATH1909) [reply code/modules/vehicles/cars/car.dm]: I mean, it won't actually affect how much damage you deal by ramming someone with your car, but still.

Is there anyway to harvest the mats from your car?

Are non-plasmamen who happen to roll uranium as their car material kind of screwed?

Are cars made of Obamium immune to damage from most melee attacks?

File: code/modules/vehicles/cars/car.dm (hunk 6 above)
Comment (cacogen) [reply code/modules/vehicles/cars/car.dm]: can't you have this affected by the material's force multiplier?

File: code/modules/vehicles/cars/car.dm [hunk 7]
```diff
@@ -93,3 +96,116 @@
...
+	. = ..()
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(!(L.mobility_flags & MOBILITY_STAND) && !L.buckle_lying)
+		for(var/i in return_drivers())
+			if(!ishuman(i))
+				continue
+			var/mob/living/carbon/human/boomer = i
+			boomer.say(pick("DON'T LIE IN FRONT OF MY CAR YA FUCKIN' KNOB!!", "HEY! I'M DRIVIN' HERE!!", "YOU ARE RUINING MY SUSPENSION, CUNT!"), forced="car crash")
```
Comment (Iamgoofball) [inline code/modules/vehicles/cars/car.dm]: No sexist slurs on our family friendly Github repo please

**Comment by Iamgoofball** [comment]
This fucking rules

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

File: code/modules/vehicles/cars/car.dm (hunk 6 above)
Comment (ATH1909) [reply code/modules/vehicles/cars/car.dm]: > can't you have this affected by the material's force multiplier?

hm yes today I will one-shot people with my adamantine car

File: code/modules/vehicles/cars/car.dm (hunk 7 above)
Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: shut the fuck up goof

Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: if someone dents my car's paint I will call them whatever word i want

File: code/modules/vehicles/cars/car.dm (hunk 6 above)
Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: no else you can fucking murder people in milliseconds. this is already strong as hell

File: code/modules/vehicles/cars/car.dm (hunk 5 above)
Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: no to all but yes to uranium. you can also blow up plasma cars

Comment (Qustinnus) [reply code/modules/vehicles/cars/car.dm]: plasma cars == trucks of peace

File: code/modules/vehicles/cars/car.dm (hunk 7 above)
Comment (cacogen) [reply code/modules/vehicles/cars/car.dm]: it's obviously cunt in the non-american sense

File: code/modules/vehicles/cars/car.dm (hunk 6 above)
Comment (cacogen) [reply code/modules/vehicles/cars/car.dm]: you can just change the default so the stronger materials aren't overpowered even for april fools

**Comment by Tlaltecuhtli** [comment]
can you make cars release co2? we need to test car engines in closed ambients

**Comment by cacogen** [comment]
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File: code/modules/vehicles/cars/car.dm (hunk 7 above)
Comment (Iamgoofball) [reply code/modules/vehicles/cars/car.dm]: doesn't matter, this is the github, you need to abide by Github's ToS @Qustinnus

**Pull request closed without merging**